src/native/corehost/fxr/fx_muxer.cpp

```diff
@@ -857,6 +857,36 @@ namespace
         g_context_initializing_cv.notify_all();
         return rc;
     }
+
+    int get_runtime_delegate_validate(const host_context_t *context, coreclr_delegate_type type)
```

Comment:
nit: I'd prefer `validate_runtime_delegate_type` or `validate_delegate_type`

Reply:
I've recombined the variants back into `get_runtime_delegate`, so this one is gone.